### DISCUSSION TITLE
考虑支持虎牙小程序吗?(#^.^#)

### DISCUSSION THREAD

**Discussion by darkfiredarkhalo** [NervJS/taro#10999] [Q & A]
虎牙小程序基于react-native-web, 使用react-native开发, [文档](https://dev.huya.com/docs#/miniapp/dev/index)  
考虑支持下吗?(#^.^#)

**Comment by Chen-jj** [comment]
666，我们研究研究~

**Reply by LovingANA** [reply]
哈哈
